app/models/concerns/audit_extensions.rb

```diff
@@ -43,6 +44,17 @@ def self.humanize_class_name
 
   private
 
+  def filter_encrypted
+    self.audited_changes.each do |name,change|
+      next if change.nil? || change.to_s.empty?
+      if change.is_a? Array
+        change.map! {|c| c.to_s.start_with?(EncryptValue::ENCRYPTION_PREFIX) ? _("[encrypted]") : c}
+      else
+        audited_changes[name] = _("[encrypted]") if change.to_s.start_with?(EncryptValue::ENCRYPTION_PREFIX)
```

Comment:
Using `_()` in this context will store the translated message in the database, so it'll be stored in the language of whoever made the change instead of whoever is viewing it.

I think perhaps you want to use `_N('[encrypted]')` here to only extract it, store the English and then call `_(db_value)` in the helper for display purposes (when `db_value == '[encrypted]'`).